pkg/ddevapp/router_compose_template.yaml

```diff
@@ -31,6 +29,12 @@ services:
       - USE_LETSENCRYPT={{ .letsencrypt }}
         {{ end }}{{/* end if .letsencrypt */}}
       - TZ={{ .Timezone }}
+      # Unset proxy environment variables for Traefik to prevent blocking internal container connections
+      # Docker automatically sets both uppercase and lowercase variables
+      - HTTP_PROXY=
+      - http_proxy=
+      - HTTPS_PROXY=
+      - https_proxy=
```

Comment:
Yay!

Now I wonder if something like `NO_PROXY=*` would solve it all.

Reply:
Yes, `NO_PROXY=*` works too.